rust/BUILD.md

```diff
@@ -0,0 +1,134 @@
+# `evmrs` Development Instructions
+
+## Requirements
+
+- install Rust toolchain >= 1.81.0 from [here](https://rustup.rs/)
+
+## Building evmrs (shared c library)
+
+- build `evmrs` in release mode
+    ```sh
+    cargo build --release # OR run `make` or `make tosca-rust` in Tosca directory
+    ```
+- build `evmrs` with debug symbols
+    ```sh
+    cargo build --profile profiling
+    ```
+
+## Lint
+
+```sh
+cargo clippy --workspace --tests --benches --examples
+```
+
+## Doc
+
+```sh
+cargo doc --workspace --document-private-items --open
+```
+
+## Optimizations
+
+By default, `evmrs` is build with the simplest and most idiomatic version.
+All optimizations are behind feature flags.
+A list of all features can be found in the `[features]` section in [Cargo.toml](./Cargo.toml).
+For convenience there is also a feature named `performance` which enables all other features that improve overall performance.
+
+Most `cargo` commands accept the `--features` flag followed by a list of features to enable, e.g.
+```sh
+cargo build --features mimalloc,stack-array
+```
+
+## Testing
+
+- Rust tests
+    ```sh
+    cargo test
+    ```
+- Go tests
+    ```sh
+    cargo build --release # OR run `make` or `make tosca-rust` in Tosca directory
+    go test ../go/...
```

Comment:
This command will run lots of tests which do not concern evmrs, this should be narrowed down.

Reply:
I guess `go test ../go/interpreter/evmrs/...` would be enough. However the go integration is not merged yet (https://github.com/Fantom-foundation/Tosca/pull/778) so those files do not yet exist.